### PR TITLE
Add endpoint for all connections

### PR DESCRIPTION
**Pull request by abdullah1308**
**Notes for Reviewers**

This PR fixes #

Added GET `/api/integrations/connections`

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: server/models/connections.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type Connection struct {
 
 // swagger:response ConnectionPage
 type ConnectionPage struct {
-	Connection []Connection `json:"connection"`
+	Connections []Connection `json:"connections"`
```
Comment (MUzairS15) [inline server/models/connections.go:42]: @abdullah1308 This will also require changes in UI, right?

**Review by MUzairS15 — DISMISSED** [review]

File: server/models/connections.go (hunk 1 above)
Comment (abdullah1308) [reply server/models/connections.go:42]: @MUzairS15 These endpoints aren't being used anywhere.

**Comment by leecalcote** [comment]
Merge conflict..

**Comment by leecalcote** [comment]
Spreadsheet updated?

**Comment by leecalcote** [comment]
@senali-d is this what you need?

**Comment by abdullah1308** [comment]
Fixed merge conflict

**Comment by senali-d** [comment]
> @senali-d is this what you need?

Yes I'm waiting for this.

**Review by nebula-aac — APPROVED** [review]
LGTM

**Pull request merged**